interface-elements-for-web/articles/grid/customization-dialog/grouping-page.md

```diff
@@ -4,10 +4,12 @@ author: Anna Kondratova
 legacyId: 118821
 ---
 # Grouping Page
-Select a column in the "Columns to Group" list to group grid data.  The grid data can be grouped by more than one column, and a list of the grouped columns is displayed at the top of **Grouping** page.
+The **Grouping** page enables you to group grid data and change the order of grouping columns within the grid.
+
+Select a column's check box in the **Columns to Group** list to group grid data against this column.  The grid data can be grouped by more than one column; a list of the grouped columns is displayed at the top of the **Grouping** page.
```

Comment:
"The grid data can be grouped by more than one column; a list of the grouped columns is displayed at the top of the **Grouping** page." Why do you use a semicolon here?